snow/consensus/snowball/tree_test.go

```diff
@@ -485,23 +485,44 @@ func TestSnowballTrinary(t *testing.T) {
 	require.Equal(Red, tree.Preference())
 	require.False(tree.Finalized())
 
+	//       *
+	//     1/ \
+	//     R   *
+	//        / \
+	//       G   B
```

Comment:
What do the numbers on the edges mean? It seems like they are referencing Snowball counters but we are using snowflake here and not snowball.